### PR TITLE
Create model monitor with default email notification sent on Job fail…

### PR DESCRIPTION
**Pull request by pdhotems**
…ed and job completed

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/ml/azure-ai-ml/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ### Features Added
   - Update TLS version from 1.0 to 1.2
 ### Bugs Fixed
+  - Model monitor semd email notification issue
```
Comment (jayesh-tanna) [inline sdk/ml/azure-ai-ml/CHANGELOG.md:8]: typo?

**Comment by jayesh-tanna** [comment]
do we need any public doc/wiki updates related to this change?

**Review by PratibhaShrivastav18 — APPROVED** [review]
Let's try to have more informative PR descriptions going forward :)

**Review by jayesh-tanna — APPROVED** [review]

**Comment by pdhotems** [comment]
> do we need any public doc/wiki updates related to this change?

no Jayesh it is not needed.

**Review by kingernupur — APPROVED** [review]

**Pull request merged**